crates/core/src/db/db_metrics/mod.rs

```diff
@@ -59,6 +59,11 @@ metrics_group!(
         #[help = "The number of rows deleted from a table"]
         #[labels(txn_type: TransactionType, database_id: u64, reducer_id: u64, table_id: u32)]
         pub rdb_num_rows_deleted: IntCounterVec,
+
+        #[name = spacetime_num_rows_read]
+        #[help = "The number of rows read from a table"]
```

Comment:
I'm a tiny bit curious about the wording here, because the PR description says 'scan' and I wondered if there are any (even nitpicky) reasons to prefer scan vs read.